### DISCUSSION TITLE
Fault in the model after opening X-Y hole size compensation

### DISCUSSION THREAD

**Discussion by Davidandan** [OrcaSlicer/OrcaSlicer#7744] [General]
I have a problem when slicing with the model after opening X-Y hole size compensation.
I can't find why this happens!
Somehow when there is a horizontal hole in the model, there is an extra wall layer generated on that level!
It only happens with X-Y hole size compensation!
Have anyone had this problem?
How did you solve this?
Maybe it is a slicer issue ?

**Comment by Davidandan** [comment]
![Capture d’écran 2024-12-14 à 18 50 28](https://github.com/user-attachments/assets/38dd290e-0856-4ec9-9eb3-488d85dd1ba8)

**Comment by pgifford** [comment]
Maybe including your 3mf file or a link to the model + your settings would be helpful to those interested in helping you.

**Comment by Davidandan** [comment]
Sure, here is the file, if you want to check the settings.
The inside dimension of the tube is therefore not precise, the inside wall should be smooth !
Thank's in advance.
[Glissière Crochets V2.3mf.zip](https://github.com/user-attachments/files/18139632/Glissiere.Crochets.V2.3mf.zip)

**Comment by Davidandan** [comment]
I think i just found something.....
The x-y hole compensation only applies on the actual vertical hole! 
But when there is a horizontal hole, the wall on the inside is actually considered as an outer wall.
There is a contradiction between the two walls hence the dimensional difference on the inside.
If i set the x-y hole compensation to 0,3 makes the hole bigger only on the part of the inner where the wall is closed, but it does not apply to the part of the inner wall where there is a horizontal hole in the wall. As soon as the horizontal hole is closed it resumes and apply the x-x hole compensation again.
If i also set the x-y contour compensation to 0,3 the problem dissapears on the inside, but then my outside wall is out of dimension !
The problem seems unsolvable in Orca, unless someone has a solution, i have not found yet!
It would be frustration to have to change all the dimensions in the CAD file!

**Reply by MrBill57** [reply]
I have the same issue. I like the X-Y hole comp, but it is useless here.
The midifiers do not have that setting enabled - if they did, then X-Y hole could applied to sections only...
